Funcky.Test/Monads/OptionTest.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace Funcky.Test.Monads
 {
-    public class OptionTest
+    public sealed partial class OptionTest
```

Comment:
if we seal tests now, there are few more classes to seal

Reply:
https://github.com/polyadic/funcky/pull/222